2021-11-25 00:10:17,654 INFO    MainThread:52017 [wandb_setup.py:_flush():71] setting env: {}
2021-11-25 00:10:17,654 INFO    MainThread:52017 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-25 00:10:17,655 INFO    MainThread:52017 [wandb_init.py:_log_setup():371] Logging user logs to /home/snamburi/Debarshi/wandb/run-20211125_001017-3cv9uayo/logs/debug.log
2021-11-25 00:10:17,655 INFO    MainThread:52017 [wandb_init.py:_log_setup():372] Logging internal logs to /home/snamburi/Debarshi/wandb/run-20211125_001017-3cv9uayo/logs/debug-internal.log
2021-11-25 00:10:17,655 INFO    MainThread:52017 [wandb_init.py:init():404] calling init triggers
2021-11-25 00:10:17,655 INFO    MainThread:52017 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-25 00:10:17,655 INFO    MainThread:52017 [wandb_init.py:init():449] starting backend
2021-11-25 00:10:17,655 INFO    MainThread:52017 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-25 00:10:17,690 INFO    MainThread:52017 [backend.py:ensure_launched():199] starting backend process...
2021-11-25 00:10:17,713 INFO    MainThread:52017 [backend.py:ensure_launched():204] started backend process with pid: 52027
2021-11-25 00:10:17,714 INFO    MainThread:52017 [wandb_init.py:init():458] backend started and connected
2021-11-25 00:10:17,716 INFO    MainThread:52017 [wandb_init.py:init():519] updated telemetry
2021-11-25 00:10:17,718 INFO    MainThread:52017 [wandb_init.py:init():549] communicating current version
2021-11-25 00:10:18,565 INFO    MainThread:52017 [wandb_init.py:init():554] got version response 
2021-11-25 00:10:18,565 INFO    MainThread:52017 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-25 00:10:18,781 INFO    MainThread:52017 [wandb_init.py:init():592] starting run threads in backend
2021-11-25 00:10:20,263 INFO    MainThread:52017 [wandb_run.py:_console_start():1816] atexit reg
2021-11-25 00:10:20,264 INFO    MainThread:52017 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-25 00:10:20,265 INFO    MainThread:52017 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-25 00:10:20,267 INFO    MainThread:52017 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-25 00:10:20,267 INFO    MainThread:52017 [wandb_init.py:init():619] run started, returning control to user process
2021-11-25 00:10:20,268 INFO    MainThread:52017 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': False}
2021-11-25 00:10:21,229 INFO    MainThread:52017 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-25 00:10:21,230 INFO    MainThread:52017 [wandb_run.py:_restore():1758] restore
2021-11-25 00:10:23,540 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 543
  total_bytes: 543
}

2021-11-25 00:10:23,644 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 543
  total_bytes: 543
}

2021-11-25 00:10:23,747 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 543
  total_bytes: 543
}

2021-11-25 00:10:23,875 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 543
  total_bytes: 74158
}

2021-11-25 00:10:23,977 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 543
  total_bytes: 74158
}

2021-11-25 00:10:24,081 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}

2021-11-25 00:10:24,184 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}

2021-11-25 00:10:24,287 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}

2021-11-25 00:10:24,390 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}

2021-11-25 00:10:24,607 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}

2021-11-25 00:10:24,869 INFO    MainThread:52017 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 74158
  total_bytes: 74158
}
local_info {
}

2021-11-25 00:10:26,039 INFO    MainThread:52017 [wandb_run.py:_append_files():2184] logging synced files
